Joyce/index.js

```diff
@@ -0,0 +1,103 @@
+//1. The Greater Numbers. Create a function which accepts two arguments: the first argument being an array of numbers, and the second argument being a number. The function should return the elements of the array which are greater than the second argument.
+function findGreatest(array, num) {
+  for (let item of array) {
+    if (item > num) {
+      return item;
+    }
+  }
+}
+console.log(findGreatest([3, 4, 5], 4));
+console.log(findGreatest([10, 20, 30], 12));
```

Comment:
Task 1 ==> If I try to print I just get 20 from the second console.log() instead of both bigger numbers which are  20 and 30